docker/oms-discounts/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:10
```

Comment:
why not replacing the current dockerfile?

Reply:
thought you still could use nodemon, idk how fundamental that is tho

Reply:
not at all, I'd rather keep 1 dockerfile, what do you think?

Reply:
fine for me, I will add `.dockerignore` file to avoid copying .git and node_modules, and remove Dockerfile.dev

Reply:
seems nice to me as well.